### DISCUSSION TITLE
TwiML Bins Icon

### DISCUSSION THREAD

**Discussion by kateschwarz** [twilio-labs/paste#831] [Request: Icon] [answered]
Hi team!
I'm working through the final pieces of the new Console Navigation sidebar structure and noticed there isn't an icon in Paste for TwiML Bins. Is that something you all can provide or do I need to get an SVG from Design or Branding?

**Comment by richbachman** [comment, marked as answer]
Hi @kateschwarz 

The TwiML Bins icon will need to come from the brand team. 

Once you have it, feel free to add it to the Paste Icon library. You can find instructions on how to do that here: https://paste.twilio.design/icons/how-to-add-an-icon

Let us know if you have any questions!

**Reply by kateschwarz** [reply]
Thanks @richbachman!
